api/layer/neofs/tree_service.go

```diff
@@ -0,0 +1,32 @@
+package neofs
```

Comment:
Why did we mode tree interface to `neofs` package? It is used in the `layer` package I think it should be there